keyboards/tweetydabird/lotus58/blok/config.c

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 Markus Knutsson (@TweetyDaBird)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#define I2C_DRIVER I2CD0
+//using raw pin values, blok converter doesn't catch this
+#define I2C0_SDA_PIN 17U
+#define I2C0_SCL_PIN 16U
```

Comment:
So, how would I enable both I2C controllers?

Reply:
ALso this is named `config.c` and not `config.h`, so cant have been tested at all.

Reply:
Well, unfortunately it's a cobble together, since I have no blok, and trying to fix things for a builder of my keyboard using one. So it does compile, and according to him it worked, vs not working before.

Reply:
> So it does compile, and according to him it worked, vs not working before.

I can assure you, with the code in this PR it would not have.

Reply:
Oh, I'm not going to argue the point. I'm more interested in figuring out the I2C controller.

From
`QMK_PM2040` mcuconf.h
```
#define RP_I2C_USE_I2C0                     FALSE
#define RP_I2C_USE_I2C1                     TRUE
```
Is that is?

Reply:
I saw your PR. You sir, are a rockstar!